pkg/utils/options.go

```diff
@@ -38,6 +41,13 @@ type ControllerOptions struct {
 	// settings for the proxy server to use when communicating with the apiserver.
 	ClientConnection componentbaseconfig.ClientConnectionConfiguration
 
+	// Controllers is the list of controllers to enable or disable
+	// '*' means "all enabled by default controllers"
+	// 'foo' means "enable 'foo'"
+	// '-foo' means "disable 'foo'"
+	// first item for a particular name wins
+	Controllers []string
```

Comment:
This field seems to only work with `clusternet-controller-manger`. Thus I don't think this is the right place to put it here.